stock_picking_package_info/models/product_packaging_template.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2015 AvanzOsc
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+from openerp.addons import decimal_precision as dp
+
+
+class ProductPackagingTemplate(models.Model):
+    _name = 'product.packaging.template'
+    _description = 'Product Packaging Template'
+    _rec_name = 'packaging_template_name'
+
+    packaging_template_name = fields.Char(required=True)
+    type = fields.Selection([
```

Comment:
Builtin methods are technically a part of the `__builtins__` namespace though, so aren't reserved in all namespaces like keywords.
'
That said, I'm good with the more descriptive term. I think `package_type` might fit better in the context though, but we can leave that to @t3ddftw to figure out 😛